Tests/Algorithm/Framework/Risk/MaximumDrawdownPercentPerSecurityTests.cs

```diff
@@ -32,11 +32,9 @@ public class MaximumDrawdownPercentPerSecurityTests
         [TestCase(Language.CSharp, 0.1, false, 0, 0, false)]
         [TestCase(Language.CSharp, 0.1, true, -50, 1000, false)]
         [TestCase(Language.CSharp, 0.1, true, -100, 1000, false)]
-        [TestCase(Language.CSharp, 0.1, true, -150, 1000, true)]
         [TestCase(Language.Python, 0.1, false, 0, 0, false)]
         [TestCase(Language.Python, 0.1, true, -50, 1000, false)]
         [TestCase(Language.Python, 0.1, true, -100, 1000, false)]
-        [TestCase(Language.Python, 0.1, true, -150, 1000, true)]
```

Comment:
Why removing this test cases for the other risk management model, shouldn't be affecting it?